core/src/test/java/org/springframework/security/access/intercept/method/MockMethodInvocation.java

```diff
@@ -15,15 +15,23 @@
  */
 package org.springframework.security.access.intercept.method;
 
+import org.aopalliance.intercept.MethodInvocation;
+
 import java.lang.reflect.AccessibleObject;
 import java.lang.reflect.Method;
 
-import org.aopalliance.intercept.MethodInvocation;
-
 @SuppressWarnings("unchecked")
 public class MockMethodInvocation implements MethodInvocation {
 	private Method method;
 	private Object targetObject;
+	private Object[] arguments = new Object[0];
+
+	public MockMethodInvocation(Object targetObject, Class clazz, String methodName, Class[] parameterTypes,
+								Object[] arguments) throws NoSuchMethodException {
+		this.method = clazz.getMethod(methodName, parameterTypes);
```

Comment:
Is there any reason you aren't calling `this(targetObject, clazz, methodName, parameterTypes)`?

If not, would you mind changing this? I realize it's a minor point, but it's one less thing to keep track off.